proto/structured.proto

```diff
@@ -104,3 +105,220 @@ message CreateTableResponse {
 // service SchemaService {
 //   rpc CreateTable (CreateTableRequest) returns (CreateTableResponse);
 // }
+
+// TableRequestHeader is supplied with every structured API request.
+message TableRequestHeader {
```

Comment:
RequestHeader contains keys and raft id and replicas and so I thought we can have our own request header